ckanext/canada/tables/admin_aircraft.yaml

```diff
@@ -0,0 +1,330 @@
+    obligation: Mandatory
+    excel_required: true
+    form_required: true
+    format_type: Date (Please format the data as YYYY-MM-DD)
+    validation: This field must not be empty
+    visible_to_public: Yes
+    occurrence: Single
+    datastore_type: date
+    form_snippet: scheming/form_snippets/date.html
+
+  # 3.7
+  - datastore_id: locations_en
+    label:
+      en: Location(s) visited (English)
+      fr: Endroit(s) visité (anglais)
+    description:
+      en: To include names of all places visited to conduct government business during the trip in question, in English.
+      fr: Cette zone indique, en anglais, le nom de tous les lieux qui ont été visités dans le cadre du voyage en question réalisé pour le compte du gouvernement.
+    obligation: Mandatory
+    excel_required: true
+    form_required: true
+    format_type: Free Text (Note, use full city/region, province/state and country name where possible. Seperate multiple locations with semicolons.)
+    validation: This field must not be empty
+    visible_to_public: Yes
+    occurrence: Repeatable
+    datastore_type: text
+    form_snippet: scheming/form_snippets/textarea.html
+    form_attrs:
+      style: "width: 100%; display: block"
+
+# 3.8
+  - datastore_id: locations_fr
+    label:
+      en: Location(s) visited (French)
+      fr: Endroit(s) visité (français)
+    description:
+      en: To include names of all places visited to conduct government business during the trip in question, in french.
+      fr: Cette zone indique, en français, le nom de tous les lieux qui ont été visités dans le cadre du voyage en question réalisé pour le compte du gouvernement.
+    obligation: Mandatory
+    excel_required: true
+    form_required: true
+    format_type: Free Text (Note, use full city/region, province/state and country name where possible. Seperate multiple locations with semicolons.)
+    validation: This field must not be empty
+    visible_to_public: Yes
+    occurrence: Repeatable
+    datastore_type: text
+    form_snippet: scheming/form_snippets/textarea.html
+    form_attrs:
+      style: "width: 100%; display: block"
+
+  # 3.9
+  - datastore_id: hours
+    label:
+      en: Number of hours flown
+      fr: Le nombre d’heures de vol
+    description:
+      en: The number of hours flown. 
+      fr: Le nombre d’heures de vol.
+    obligation: Mandatory
+    excel_required: true
+    form_required: true
+    format_type: Decimal
+    validation: This field must not be empty
+    visible_to_public: Yes
+    occurrence: Single
+    datastore_type: numeric
+    form_snippet: scheming/form_snippets/textarea.html
+    form_attrs:
+      style: "width: 100%; display: block"
+
+# 3.10
+  - datastore_id: passengers
+    label:
+      en: Passenger names
+      fr: Nom des passagers
+    description:
+      en: Full list of flight passengers.
+      fr: Liste complète des passagers du vol.
```

Comment:
You're not describing how the passenger list should be separated or sorted (if at all)